requirements.txt

```diff
@@ -10,7 +10,7 @@ fastavro>=1.0.0.post1
 python-snappy>=0.5.4
 charset-normalizer>=1.3.6
 psutil>=4.0.0
-scipy>=1.4.1
+scipy>=1.4.1,<1.11.0
```

Comment:
1.11.0 causes graph profilers to start failing tests.

we will need to investigate what is failing and why https://scipy.github.io/devdocs/release/1.11.0-notes.html#statistical-distributions
causes these failures.

Reply:
Issue created #905